src/__tests__/cql/cases/gad7/gad7_reusable_resources.yml

```diff
@@ -0,0 +1,117 @@
+gad7_reusable_resources:
+# patient resource
+- &patientResource
+  resourceType: Patient
+  name:
+    -
+      family: Skywalker
+      given:
```

Comment:
slightly strange yaml formatting; if we're changing the test data frequently, it might be worth running the yaml files through yamllint